### ISSUE TITLE
feat(patch): Toggle Sponsorblock for playing video

### ISSUE THREAD

**Issue by fatalcoder524** [revanced/revanced-patches#1907]
### Application

YouTube

### Issue

Sometimes users wrongfully flag a section of video and that section cannot be seen unless we completely turn off sponsorblock for all videos and replay it.

### Patch

Add a Toggle button or option in sponsorblock submit menu or an option in player menu to enable or disable Sponsorblock for the current playing video.

### Motivation

This will help a lot of users to easily verify if a submitted section of video is categorised properly and correct it by voting on it.

It will also help people who might like a certain section like music in video which might be auto skipped based on global settings but also you like to view music for this particular video.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by fatalcoder524** [comment]
> You can change categories to "skip automatically once", and then you can rewind and view segments.

I tried this. This worked better than expected. But the issue is for replaying the video instantly or selecting the same video from feed, it didn't skip the part automatically for the first time. I'm ok with replaying the video instantly and it not working. But selecting the video again from feed didn't skip it the first time. I had to kill the app and then it worked fine.
